src/core_atmosphere/physics/physics_wrf/module_sf_ruclsm.F

```diff
@@ -574,10 +574,6 @@ subroutine lsmruc(spp_lsm,                                   &
          enddo
      enddo
 
-        do k=1,nsl
-           soilice(k)=0.
-           soiliqw(k)=0.
-        enddo
```

Comment:
@jderrico-noaa Janet, could we revert removing lines 577-580? After all I think it is safer to have this initialization to zero.